src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -95,6 +96,19 @@ PYBIND11_MODULE(pyopenvino, m) {
         py::arg("model"),
         py::arg("batch_size") = -1);
 
+    m.def(
+        "serialize",
+        [](std::shared_ptr<ov::Model>& model,
+           const std::string& xml_path,
+           const std::string& bin_path,
+           const std::string& version) {
+            ov::serialize(model, xml_path, bin_path, Common::convert_to_version(version));
+        },
+        py::arg("model"),
+        py::arg("model_path"),  // TODO: rename to xml_path, bin_path ?
+        py::arg("weights_path"),
+        py::arg("version") = "UNSPECIFIED");
```

Comment:
You need to extend this function with docs similar to ones from offline_transformation API. Probably copying them and applying small modifications would be fine.